nixio/pycore/property.py

```diff
@@ -144,3 +144,22 @@ def __str__(self):
 
     def __repr__(self):
         return self.__str__()
+
+    def __values_to_string(self, max_length=80):
+        unit = "" if self.unit is None else self.unit
+        value_str = ""
+        for i, v in enumerate(self.values):
+            value_str += ("%s%s") % (", " if i > 0 else "", v.to_string(unit=unit))
```

Comment:
For this bit, instead of figuring out where to add a leading comma (by checking `i > 0`), it would be simpler to make a list of value strings and then just do `", ".join(vstr_list)`.

The whole thing can be: `value_str = ", ".join(v.to_string(unit=unit) for v in self.values)`